frontend/test/metabase/scenarios/README.md

```diff
@@ -2,4 +2,39 @@
 
 ## Running
 
-- If you are running tests that include `alert > email_alert`, run `python -m smtpd -n -c DebuggingServer localhost:1025`in terminal first for setting up email through your localhost
+- If you are running tests that include `alert > email_alert`, run `docker run -p 80:80 -p 25:25 maildev/maildev` in terminal first for setting up email through your localhost
+
+## Metabase QA DB Tests
+
+If you're running Cypress locally and you need to write or run tests for one of the [supported databases](https://github.com/metabase/metabase-qa), please make sure that you have `QA_DB_ENABLED` env for that session only. 
+
+### Requirements
+
+Prior to running these tests:
+
+```bash
+QA_DB_ENABLED=true
+```
+
+```fish
+set -x QA_DB_ENABLED true
+```
+
+
+You will also need to have all QA DB Docker images running. For the convenience, we'll list docker commands here as well:
+
+```shell
+#Mongo 4
+docker run --rm -p 27017:27017 --name meta-mongo-sample metabase/qa-databases:mongo-sample-4.0
```

Comment:
```suggestion
docker run --rm -p 27017:27017 --name meta-mongo4-sample metabase/qa-databases:mongo-sample-4.0
```

Reply:
Good catch, but then it needs to be updated in the QA repo as well:
https://github.com/metabase/metabase-qa#mongo-4